src/appengine/handlers/revisions_info.py

```diff
@@ -51,7 +52,8 @@ def get(self):
       raise helpers.EarlyExitException('Failed to get component revisions.',
                                        400)
 
-    self.render('revisions-info.html',
-                {'info': {
-                    'componentRevisionsList': component_revisions_list
-                }})
+    return self.render(
+        'revisions-info.html',
+        {'info': {
```

Comment:
nit:  this indentation looks a bit weird. Can you space things out a bit more suchj that this looks like:

```python
{
   'info': {
     'componentRevisionsList...':
   },
})
```